tests/dummy/lib/module-name-inliner/node_modules/ember-cli-version-checker/node_modules/.bin/semver

```diff
@@ -0,0 +1 @@
+../../../../../../../../node_modules/semver/bin/semver.js
```

Comment:
O_o

Reply:
Ummm. What.

Reply:
Removed in 4fe957e. No idea what that was about.